templates/managed-ec2-core.yaml

```diff
@@ -0,0 +1,44 @@
+Description: Allows central management of computing instances; building on essentials
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  MaintenanceWindow:
+    Description: 'Acceptable outage intervals for Systems Manager changes'
+    Type: String
+    Default: 'Weekly-0400SundayUTC-7'
+Resources:
+  ComputingInstanceProfile:
+    Type: 'AWS::IAM::InstanceProfile'
+    Properties:
+      Path: "/"
+      Roles:
+        - !Ref ComputingInstanceRole
+  ComputingInstanceRole:
```

Comment:
isn't this already in essentials.yaml? why do you need it in both files?

Reply:
I wanted it in both initially because I wanted to be able to apply it to currently deployed instances and new ones